build/BundledDotnetTools.props

```diff
@@ -1,6 +1,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
-    <BundledDotnetTools Include="dotnet-watch" Version="$(AspNetCoreVersion)" />
     <BundledDotnetTools Include="dotnet-dev-certs" Version="$(AspNetCoreVersion)" />
+    <BundledDotnetTools Include="dotnet-sql-cache" Version="$(AspNetCoreVersion)" />
+    <BundledDotnetTools Include="dotnet-user-secrets" Version="$(AspNetCoreVersion)" />
+    <BundledDotnetTools Include="dotnet-watch" Version="$(AspNetCoreVersion)" />
```

Comment:
I think we should probably add watch to the general help menu, as it is a lot more general use than any of the other ones.

cc @KathleenDollard

Reply:
By the way, we can do that in a separate PR. I am approving this one.